src/vnsw/agent/oper/interface.cc

```diff
@@ -451,7 +453,7 @@ void Interface::GetOsParams(Agent *agent) {
             "> querying mac-address for interface <" << name << "> " <<
             "Agent-index <" << id_ << ">");
         os_oper_state_ = false;
-        close(fd);
+        closesocket(fd);
```

Comment:
`closesocket` does not exist on Linux. I recommend `#define closesocket close` on non-Windows systems or something similar.